### PR TITLE
Language teams

### PR DESCRIPTION
**Pull request by phlax**
Adds a language team class and plugin getter

### CODE REVIEW

File: pootle/apps/pootle_language/teams.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+    def add_member(self, user, role):
+        permission_set = self.get_permission_set(user, create=True)
+        for permission in self.get_permissions_for_role(role):
+            permission_set.positive_permissions.add(permission)
+
+    def get_permissions_for_role(self, role):
+        ct = ContentType.objects.get_for_model(self.language.directory)
+        return Permission.objects.filter(
+            content_type=ct).filter(codename__in=self.roles[role])
```
Comment (unho) [inline pootle/apps/pootle_language/teams.py]: Is the content type filtering actually necessary?

Comment (phlax) [inline pootle/apps/pootle_language/teams.py]: probs not in the current use

File: pootle/apps/pootle_language/teams.py [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+    def get_permission_set(self, user, create=False):
+        if create:
+            permission_set, __ = (
+                self.language.directory.permission_sets.get_or_create(
+                    user=user, directory=self.language.directory.id))
+            return permission_set
```
Comment (unho) [inline pootle/apps/pootle_language/teams.py]:
```python
            return self.language.directory.permission_sets.get_or_create(
                user=user, directory=self.language.directory.id))[0]
```

**Comment by unho** [comment]
Nits aside, lgtm.

**Pull request merged**